### PR TITLE
cirrus-ci: add FreeBSD 13 and bump FreeBSD 12 to 12.3

### PR DESCRIPTION
**Pull request by clementperon**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ task:
   freebsd_instance:
     matrix:
       image_family: freebsd-11-4
-      image_family: freebsd-12-2
+      image_family: freebsd-12-3
+      image_family: freebsd-13-0
```
Comment (seladb) [inline .cirrus.yml:7]: I'm wondering if we should be running 3 versions of FreeBSD in our CI, isn't it too much?

Comment (clementperon) [reply .cirrus.yml:7]: If you agree we can drop 11-4 which is no more maintained

Comment (seladb) [reply .cirrus.yml:7]: Sure, let's do that!

Comment (clementperon) [reply .cirrus.yml]: done

**Pull request merged**